apps/calendar/style/modify_account_view.css

```diff
@@ -24,7 +24,7 @@
 }
 
 #modify-account-view.in-progress .during-progress > div {
-  display: inline-block;
+  display: block;
```

Comment:
Isn't block the default? Why do we need this? If we can remove it, I think we should.